Raywire/iReporter
# [start #163820043] Adds endpoint to request and verify a user's account

**What does this PR do?**
Adds endpoints to request an account verification
**Descriptions of the tasks to be completed?**
Create an endpoint to request an account verification link
Create an endpoint to verify a user's account
**How should this be manually tested?**
Git clone the repo and cd into it, set up the requirements as per the README file
Using Postman on your localhost 
`POST:` `http://127.0.0.1:5000/api/v2/users/<username>/requestVerification`

`PATCH:` `http://127.0.0.1:5000/api/v2/users/<username>/verify`

**Pivotal Tracker Story**
#163820043
